examples/common/text_data.py

```diff
@@ -63,12 +62,6 @@ def __init__(self,
                  num_canonical_nodes: Optional[int] = None,
                  batch_size: Optional[int] = None):
 
-        # Validation
```

Comment:
Can we keep the deprecation warning you had, except make it a deprecation error? I believe you can do this by adding `kwargs` as a init arg, and then checking in kwargs for `group_method`

Reply:
Does python have a `DeprecationError`? Or are you saying that if I `raise DeprecationWarning('no group_method 4 u')` that will act like an error?

Reply:
I think maybe just raise a `ValueError` or whatever other error you feel is appropriate, with the nice message that you wrote

Reply:
luckily the later is effectively true in that it causes streaming to crash 🎉